app/src/main/java/com/randomappsinc/studentpicker/grouping/GroupingSettings.java

```diff
@@ -2,13 +2,12 @@
 
 public class GroupingSettings {
 
+    private int listSize;
```

Comment:
nit: listSize is a little too generic for me.
```suggestion
    private int namelistSize;
```